src/doc/src/reference/specifying-dependencies.md

```diff
@@ -19,18 +19,32 @@ guide](../guide/index.md), we specified a dependency on the `time` crate:
 time = "0.1.12"
 ```
 
-The string `"0.1.12"` is a version requirement. Although it looks like a
-specific *version* of the `time` crate, it actually specifies a *range* of
-versions and allows [SemVer] compatible updates. An update is allowed if the new
-version number does not modify the left-most non-zero digit in the major, minor,
-patch grouping. In this case, if we ran `cargo update -p time`, cargo should
-update us to version `0.1.13` if it is the latest `0.1.z` release, but would not
-update us to `0.2.0`. If instead we had specified the version string as `1.0`,
-cargo should update to `1.1` if it is the latest `1.y` release, but not `2.0`.
-The version `0.0.x` is not considered compatible with any other version.
+As specified in [SemVer]
+a version identifier is composed of up to three integers separated by
+a period: `x`, `x.y`, or `x.y.z` where x is the major version, y is the minor
+version and z is the patch version.
+
+A version identifier such as the `"0.1.12"` in the above toml excerpt is interpreted
+as a [Caret requirement]. In other words, although it looks like a
+specific *version* it actually specifies a *range* of acceptable versions as described
+below.
 
 [SemVer]: https://semver.org
 
+### Caret requirement
```

Comment:
Moving and editing in the same commit is making this a lot harder to compare the old and new versions of the documentation.  Please split these out into distinct commits

I'm limiting how deeply I look at this until this is resolved.

Reply:
There are exactly two commits in the PR (AFAIK). The second commit is merely fixing two typos (spelling of caret is with one, not two r's).

So, unless there is something about the first commit that I can split out for you, please don't let the fact that there is a second commit delay your review.

Reply:
Initially, it looked like sections were re-arranged but looking back, it was that the header for the caret section was moved up.